ui/StatusQ/src/StatusQ/Controls/StatusComboBox.qml

```diff
@@ -31,6 +32,36 @@ Item {
     property int size: StatusComboBox.Size.Large
     property int type: StatusComboBox.Type.Primary
 
+    readonly property Component defaultBackgroundComponent: Rectangle {
```

Comment:
Oh, one more... we have `StatusComboboxBackground.qml` already, pls use/derive from that instead

Reply:
Do you think it's worth? I don't see much in common..

Reply:
Yeah, probably not in this case :)